pages/content/amp-dev/community/roadmap.html

```diff
@@ -2,86 +2,185 @@
-        <div class="ap-o-roadmap-filter-list all" [class]="roadmapfilter.activefilter != 'all' ? 'ap-o-roadmap-filter-list filtered' : 'ap-o-roadmap-filter-list all' ">
-          {% for label in doc.roadmap.labels %}
+    <h2 class="ap-t-roadmap-filter-headline">{{ _('Select working groups') }}</h2>
+    <div class="ap-t-roadmap-filter-list">
+      <button class="ap-m-filter-bubble reset [% if not category %]active[% endif %]"
+          style="color: #005AF0;"
+          on="tap:AMP.navigateTo(url='{{ doc.url.path }}#status-updates')">
+        <span class="ap-m-filter-bubble-text">{{ _('All') }}</span>
+      </button>
+      {% for group in doc.roadmap.working_groups %}
+      <button class="ap-m-filter-bubble {{ group.slug }} [% if category == '{{ group.slug }}' %]active[% endif %]"
+          style="color: {{ group.color }}"
+          on="tap:AMP.navigateTo(url='{{ doc.url.path }}?category={{ group.slug }}#status-updates')">
+        <span class="ap-m-filter-bubble-text">{{ group.name }}</span>
+      </button>
+      {% endfor %}
+    </div>
+  </section>
 
-          <button class="ap-m-filter-bubble {{ label | slug}} active" on="tap:AMP.setState({roadmapfilter: {activefilter: '{{ label | slug}}' } })"
-          [class]="roadmapfilter.activefilter == 'all' || roadmapfilter.activefilter == '{{label|slug}}' ? 'ap-m-filter-bubble {{ label | slug}} active' : 'ap-m-filter-bubble {{ label | slug}}'">
-            {{ _(label) }}
-            <span tabindex="-1" role="reset" class="ap-m-filter-bubble-reset" [class]="roadmapfilter.activefilter == '{{ label | slug}}' ? 'ap-m-filter-bubble-reset show': 'ap-m-filter-bubble-reset'" on="tap:AMP.setState({roadmapfilter: {activefilter: 'all' }})">
-                <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#close"></use></svg>
-            </span>
-          </button>
+  <section class="ap-t-roadmap-container">
 
-          {% endfor %}
-        </div>
-      </section>
-
-      <div class="ap-o-roadmap-tiles ap--container-fluid">
-        <!-- Columns -->
-        {% do doc.icons.useIcon('icons/notepad.svg') %}
-        {% do doc.icons.useIcon('icons/wrench.svg') %}
-        {% do doc.icons.useIcon('icons/dolly.svg') %}
-
-        {% for column in doc.roadmap.columns %}
-        <div class="ap-o-roadmap-tiles-list">
-          <div class="ap-a-ico">
-            <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#{% if loop.index == 1 %}notepad{% elif loop.index == 2 %}wrench{% elif loop.index == 3 %}dolly{% endif %}"></use></svg>
+    <div class="ap-t-roadmap-navigation">
+      <div class="ap-t-roadmap-navigation-list">
+        {% for quarter in doc.roadmap.list|map(attribute='quarter')|unique|list %}
+
+        {% set roadmap_quarter = dict(doc.roadmap.list|groupby('quarter')) %}
+        [% set roadmap_filtered = {{ dict(roadmap_quarter[quarter]|groupby('wg_slug'))|tojson }} %]
+
+        [% if not category or roadmap_filtered[category]|length > 0 %]
```

Comment:
This condition checks if there are issues for a certain working group in a certain quarter, no? If that's the case then the comment above applies and this could be simplified to:

```
[% set quarters = {{ doc.roadmap.quarters|tojson }} %]
[% if not category or quarters['{{ quarter }}'].includes(category) %]
```

64 and 65 can then be omitted and you have a lot less logic in the template.